lmcache/experimental/cache_engine.py

```diff
@@ -305,7 +305,15 @@ def retrieve(self,
                     future_memory_obj = asyncio.run_coroutine_threadsafe(
                         self.distributed_server.issue_get(key),
                         self.distributed_loop)
-                    memory_obj = future_memory_obj.result()
+                    try:
+                        memory_obj = future_memory_obj.result(1)
+                    except TimeoutError:
```

Comment:
We need to limit this timeout to something reasonable since nothing cleans the indexer at the moment. It's possible for a vLLM pod to crash and remain listed in Redis, which would cause a long timeout.

Reply:
So what do you think is the default timeout we should set?